.changeset/lazy-balloons-report.md

```diff
@@ -0,0 +1,9 @@
+---
+"@opennextjs/cloudflare": patch
+---
+
+checks and creates a wrangler.jsonc file for the user in case one wrangler.(toml|json|jsonc) file is not already present
+
+also introduce a new `--skipWranglerConfigCheck` cli flag and a `SKIP_WRANGLER_CONFIG_CHECK`
```

Comment:
I think that this makes sense, generally I expect the vast majority of people to just use standard wrangler config files, those that however do not want to do that and want to instead use a custom config location should have a way to opt out of the config check